README.md

```diff
@@ -69,6 +70,51 @@ A CMake-based project can use the headers without installing, as follows:
 See also the [example](example/) subdirectory.  But since that example is
 *inside* this repostory, it doesn't use and `add_subdirectory` directive.
 
+### Using Bazel
+A Bazel-based project can use the headers without installing, as follows:
+
+1. Add a `local_repository` to your `WORKSPACE` file. For example:
```

Comment:
I think we should mention that "First, you should add SPIRV-Header as your submodule." ... "For example, if you put it in `external/spirv-headers` as a submodule: ...".